mlflow/models/wheeled_model.py

```diff
@@ -234,6 +234,7 @@ def _download_wheels(cls, pip_requirements_path, dst_path):
                     "-r",
                     pip_requirements_path,
                     "--no-cache-dir",
+                    "--progress-bar=off",
```

Comment:
Does this work? We can use `MLFLOW_ENABLE_ARTIFACTS_PROGRESS_BAR` env var to turn it off

Reply:
Ohh never mind, this is for wheel downloading progress bar